File: app/src/main/java/com/example/bookfinderapp/modelV2/ListPrice__1.java
package com.example.bookfinderapp.modelV2;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ListPrice__1 {

    @SerializedName("amountInMicros")
    @Expose
    private Long amountInMicros;
    @SerializedName("currencyCode")
    @Expose
    private String currencyCode;

    public Long getAmountInMicros() {
        return amountInMicros;
    }

    public void setAmountInMicros(Long amountInMicros) {
        this.amountInMicros = amountInMicros;
    }

    public String getCurrencyCode() {
        return currencyCode;
    }

    public void setCurrencyCode(String currencyCode) {
        this.currencyCode = currencyCode;
    }

}

File: app/src/main/java/com/bevstudio/wolfbooksapp/model/api/ListPrice__1.kt
package com.bevstudio.wolfbooksapp.model.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ListPrice__1 {
    @SerializedName("amountInMicros")
    @Expose
    var amountInMicros: Long? = null

    @SerializedName("currencyCode")
    @Expose
    var currencyCode: String? = null
}